webroot/adm/client-key.html

```diff
@@ -87,6 +94,14 @@ <h3>Output</h3>
             doApiCall('GET', '/api/client/keyId?keyId=' + $('#searchByKeyId').val(), '#standardOutput', '#errorOutput');
         });
 
+        $('#searchContact').on('click', function () {
+            if(!$('#searchByContact').val()) {
+                $('#errorOutput').html("Required Parameters: key_id")
```

Comment:
shouldn't this be `Required Parameters: Contact`?

Reply:
Yeah it should 🤦